gateway-service-definitions/src/main/resources/services/yarnui/2.7.0/rewrite.xml

```diff
@@ -23,8 +23,8 @@
      e.g. http://host.com:8088
 -->
 
-<rule dir="IN" name="YARNUI/yarn/inbound/ws" pattern="*://*:*/**/yarn/ws/v1/{**}">
-    <rewrite template="{$serviceUrl[YARNUI]}/ws/v1/{**}"/>
+<rule dir="IN" name="YARNUI/yarn/inbound/ws" pattern="*://*:*/**/yarn/ws/v1/{**}?{**}">
```

Comment:
Does this still work WITHOUT any query params?

Reply:
I'd add a new rule for `{**}?{**}` case then updating the existing one.

Reply:
> Does this still work WITHOUT any query params?

yes, without `?{**}`, the query params are ignored

Reply:
Yes, I have test Yarn RM API
`/ws/v1/cluster/apps` works! and `/ws/v1/cluster/apps?states=running` works

[Cluster Applications API](https://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-site/ResourceManagerRest.html#Cluster_Applications_API)

Reply:
> I'd add a new rule for `{**}?{**}` case then updating the existing one.

afaik It's not necessary to do that.